config/sync/core.entity_view_display.media.embeddable.media_library.yml

```diff
@@ -0,0 +1,54 @@
+uuid: e17dc88d-5714-48a8-ad06-a0eaaa78d27a
+langcode: en
+status: true
+dependencies:
+  config:
+    - core.entity_view_mode.media.media_library
+    - field.field.media.embeddable.field_media_embeddable_code
+    - field.field.media.embeddable.field_media_embeddable_oembed
+    - image.style.medium
+    - media.type.embeddable
+  module:
+    - image
+    - media
+id: media.embeddable.media_library
+targetEntityType: media
+bundle: embeddable
+mode: media_library
+content:
```

Comment:
Hide everything except the thumbnail and the name.

Reply:
fixed!